### PR TITLE
Update smartrader style

### PR DESCRIPTION
**Pull request by ghost**
#### Changelog

- Updated the style as per design.
- Updated the icons as per design.

### CODE REVIEW

File: src/sass/_common/base/constants.scss [hunk 1]
```diff
@@ -45,6 +45,7 @@ $COLOR_NETWORK_STATUS_BLINKER: #4bb4b3;
 $COLOR_BORDER_GRAY: #f2f3f4;
 $COLOR_BALLOON_GRAY: #d6dadb;
 $COLOR_HOVER_FULLSCREEN_GRAY: #e6e9e9;
+$COLOR_CONTRACT_ERROR_YELLOW: #fff2df;
```
Comment (aaimio) [inline src/sass/_common/base/constants.scss]: Could we add this to `src/sass/_common/base/constants-deriv.scss`?

File: src/sass/_common/common.scss [hunk 2]
```diff
@@ -46,7 +46,7 @@ table {
 
 select, input, button, textarea {
     background: $COLOR_WHITE;
-    border: 1px solid $COLOR_GRAY;
+    border: 1px solid $COLOR_BALLOON_GRAY;
```
Comment (aaron-binary) [inline src/sass/_common/common.scss]: You can use `var(--general-active)` from `src/sass/_common/base/theme.scss`.

File: src/sass/_common/jquery_elements.scss [hunk 3]
```diff
@@ -9,18 +9,81 @@
     }
 }
 
+#ui-timepicker-div {
+    display: flex;
+}
+
 #ui-timepicker-div, #ui-datepicker-div {
+    border: 2px solid $COLOR_BALLOON_GRAY;
+    border-radius: 4px;
```
Comment (aaimio) [inline src/sass/_common/jquery_elements.scss:26]: Can use `$BORDER_RADIUS` here.

**Comment by aaimio** [comment]
Please update your styles to use CSS vars from `theme.scss`.

**Pull request merged**